content/3_Silver/Prefix_Sums_2.mdx

```diff
@@ -102,11 +102,102 @@ export const problems = {
 
 <FocusProblem problem={problems.maxsum} />
 
+### Solution
+
+<Spoiler title="Solution">
+
+Consider the prefix sum array $p$. The subarray sum $a_j \dots a_{i-1}$ where $j\leq i$ is $p[i]-p[j]$, so we are looking for the maximum possible value of $p[i]-p[j]$ over $0 \leq i \leq j \leq N$. 
```

Comment:
use \texttt for variables

Reply:
not one letter vars tho

Reply:
Can you commit a suggestion, so that ik what you're talking ab??

Reply:
i think the `p[i]` and `p[j]`, right?

Reply:
yes use \texttt{p}[i] etc

Reply:
I think `\texttt{p[i]}` looks better? idk
![image](https://user-images.githubusercontent.com/63619830/111053264-339d6780-8417-11eb-9479-83f3c27c2175.png)

Reply:
hmm i like the 2nd one but the 2nd one is standard anyways